tuning/trainercontroller/operations/logcontrol.py

```diff
@@ -27,6 +29,10 @@ def __init__(self, log_format: str, log_level: str, **kwargs):
                 "Specified log_level [%s] is invalid for LogControl" % (log_level)
             )
         self.log_format = log_format
+        self.final_log_format = DefaultFinalLogMessage
```

Comment:
Is there a reason the final log format change needs a change in the log control? If users want they should just use two different sections of TC based on different event type, isn't it?